app/partials/project-card.cjsx

```diff
@@ -31,15 +34,24 @@ ProjectCard = React.createClass
     else
       '/projects/' + @props.project.slug
 
-    <FlexibleLink to={href}>
-      <div className="project-card" ref="projectCard" style={conditionalStyle}>
+    cardBody =
+      <div className="project-card #{this.props.customCSS}" ref="projectCard" style={conditionalStyle}>
         <svg viewBox="0 0 2 1" width="100%"></svg>
-        <div className="details">
+        <div className="details" style={{cursor: "default"}}>
           <div className="name"><span>{@props.project.display_name}</span></div>
+          {<hr /> if @props.landingPage}
           {<div className="description">{@props.project.description}</div> if @props.project.description?}
           <button type="button" tabIndex="-1" className="standard-button card-button"><Translate content={"projectsPage.button"} /></button>
+          {if @props.landingPage
+            <Link to={href} className="primary-button" type="button">View Project</Link>}
         </div>
       </div>
-    </FlexibleLink>
+
+    if !@props.landingPage
+      <FlexibleLink to={href}>
+        {cardBody}
+      </FlexibleLink>
+    else
+        cardBody
```

Comment:
I think the cards themselves are on a different PR. Elsewhere on the site, the entire card is a link, but on the home page there's a button-style link to click.

It might not be a big deal, but I'd got used to just clicking on the project avatar, so this threw me a little bit.